README.md

```diff
@@ -1,56 +1,119 @@
-# Getting Started with Cisco IOS-XE Smart licensing APIs
+# Getting Started with IOS-XE SLP REST APIs
 
-Sample code in Python that shows how to call IOS-XE SLP REST APIs. You can automate the usage reporting workflows using these APIs.
 
-For information about Smart Licensing Using Policy (SLP), see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).
+Sample Python script written in Python 3.8.5. The script demonstrates the use of IOS-XE SLP REST APIs to automate Smart Licensing Using Policy (SLP) workflows.
+
+SLP was first introduced on Cisco IOS-XE product instances in release Cisco IOS-XE Amsterdam 17.3.2. For more information, see [About Smart Licensing Using Policy](https://developer.cisco.com/docs/ios-xe/smart-licensing/#!introduction/about-smart-licensing-using-policy).
 
 ## Use Case Description
 
-For information about the purpose of each currently available API, see [About IOS-XE SLP REST APIs](ios-xe-smart-licensing-api/intro.md#about-ios-xe-slp-rest-apis).
+For information about each API that is demonstrated in this script, see [API Endpoint and Description](https://developer.cisco.com/docs/ios-xe/smart-licensing/#!about-the-apis/api-endpoint-and-description).
 
 Future enhancements include an API to return a Smart Licensing Authorization Code (SLAC).
 
 ## Installation
 
-Python must be installed. Further the following Python packages are required:
-* os
-* shutil
-* json
-* xmltodict
-* requests
-* yaml
-* tarfile
+Install [Python](https://www.python.org/downloads/) and the required packages. 
+
+* Python version: 3.8.5
+* Required Python packages:
+  * os
+  * shutil
+  * json
+  * xmltodict
+  * requests
+  * yaml
+  * tarfile
+  
 
 ## Configuration
-Configure the following where? we have to check CSSM API docs to know this and then add. 
 
-client_id: 55f65e59-df52-4073-bb12-4268d5167019 (the user has to enter their own;  will enquire how this should be done)
-client_secret: 78962244-9f2f-4e50-b233-f92adedd1e33 (the user has to enter their own; will enquire how this should be done)
-username: CSSM log in username (update config.yaml with dummy username)
-password: CSSM log in password (update config.yaml with dummy password)
+
+Configure the following in the ```config.yaml``` file: 
+
+1. In the ```api_keys:``` section enter the username and password that you use to log in to the CSSM Web UI. 
+
+	```
+    api_keys:
+  
+    	client_id: not required
```

Comment:
We have to change this as well, because now the option 1 to get SA/VA is working.

Reply:
Done.  Please see the updated draft.